zenoh/tests/shm.rs

```diff
@@ -128,6 +125,7 @@ async fn test_session_pubsub(peer01: &Session, peer02: &Session, reliability: Re
             .declare_subscriber(&key_expr)
             .callback(move |sample| {
                 assert_eq!(sample.payload().len(), size);
+                assert!(sample.payload().deserialize::<&zshm>().is_ok());
```

Comment:
In this particular case it won't make much difference but in the tests is better to use `unwrap()` since upon failure the actual error will be printed out.

Reply:
Agree, fixed